rmgpy/scoop_framework/util.py

```diff
@@ -160,6 +172,8 @@ def submit_(func, *args, **kwargs):
     returns the return value of the called function, or
     when SCOOP is loaded, the future object.
     """
+    warnings.warn("The option scoop is no longer supported"\
+     " and may be removed in Version: 2.3 ", DeprecationWarning)
```

Comment:
For all of these warnings, could you indent the second line to line up with the " in the first line? Also, the slashes shouldn't be necessary.

Reply:
Done. Got the template from here:
https://github.com/ReactionMechanismGenerator/RMG-Py/wiki/RMG-Contributor-Guidelines
I guess it should be changed accordingly?